Gaijinizer patch/CommonEvents_238.txt

```diff
@@ -128,102 +141,114 @@ What……!!What……!!Mm, Muk...Roh ...!?#MTLed
 僕はもう、恐怖と快感の入り交じりで
 泣き出しそうになっていた……
 > CONTEXT: CommonEvents_238/99/Dialogue
-While both teats are stimulated by them and pant ...
-I am already mixed with Afraid and pleasure
-I was about to cry ...#MTLed
+Both my nipples were being stimulated by those two...
+While breathing raggedly, I felt a mix of fear and
+pleasure, I was about to cry...#Ron
```

Comment:
Flow:
>_**Both my nipples were being stimulated by those two...
While breathing raggedly, I felt a mix of fear and
pleasure. I was about to cry...#Ron**_